python/packages/autogen-core/src/autogen_core/models/_types.py

```diff
@@ -51,6 +51,7 @@ class FunctionExecutionResult(BaseModel):
 
     content: str
     call_id: str
+    is_error: bool
```

Comment:
Let's make this field optional and default to false to not introduce a breaking change.